protocol/src/index.ts

```diff
@@ -651,6 +651,21 @@ export namespace Protocol {
     '|controlshtml|': readonly ['controlshtml', HTML];
     '|fieldhtml|': readonly ['fieldhtml', HTML];
     '|debug|': readonly ['debug', Message];
+    '|deinit|': readonly ['deinit'];
+    '|selectorhtml|': readonly ['selectorhtml', string, HTML];
+    '|refresh|': readonly ['refresh'];
+    '|tempnotify|': readonly ['tempnotify', string, Message, ...[] | [Message] | [Message, string]];
+    '|tempnotifyoff|': readonly ['tempnotifyoff', string];
+    '|noinit|namerequired|': readonly ['noinit', 'namerequired', Message];
+    '|noinit|nonexistent|': readonly ['noinit', 'nonexistent', Message];
+    '|noinit|joinfailed|': readonly ['noinit', 'joinfailed', Message];
+    '|noinit|rename|': readonly ['noinit', 'rename', string, string];
+    '|hidelines|delete|': readonly ['hidelines', 'delete', string, Num];
+    '|hidelines|hide|': readonly ['hidelines', 'hide', string, Num];
+    '|hidelines|unlink|': readonly ['hidelines', 'unlink', string];
+    '|expire|': readonly ['expire', Message?];
+    '|modaction|': readonly ['modaction', Message];
```

Comment:
@mia-pi-git has suggested this one might be internal-only and never actually sent over the wire?

Reply:
Yea, I don't see it anywhere in the client code. I'll remove that one then.